src/app/modules/account/containers/signin/signin.component.html

```diff
@@ -10,7 +10,10 @@ <h1>Zaloguj się</h1>
 
     <mat-form-field appearance="fill" class="form-control">
       <mat-label>Hasło</mat-label>
-      <input matInput type="password" placeholder="Hasło" formControlName="password" required>
+      <button mat-icon-button matSuffix (click)="hide = !hide" [attr.aria-label]="'Hide password'" [attr.aria-pressed]="hide">
```

Comment:
pls create function for click event
```suggestion
      <button mat-icon-button matSuffix (click)="toggleHide()" [attr.aria-label]="'Hide password'" [attr.aria-pressed]="hide">
```
and
```suggestion
      <button mat-icon-button matSuffix (click)="hide = !hide" aria-label="Hide password" [attr.aria-pressed]="hide">
```